consoles/vnc_base.pm

```diff
@@ -66,7 +66,9 @@ sub current_screen ($self) {
     return $self->{vnc}->_framebuffer;
 }
 
-sub _typing_limit () { $bmwqemu::vars{VNC_TYPING_LIMIT} // VNC_TYPING_LIMIT_DEFAULT || 1 }
+sub _typing_limit () {
```

Comment:
It seems this subroutine is never used anywhere. Maybe we should remove it instead.

Reply:
Also the #2590 subroutine was never used, but we decided to keep that. I've not strong opinion whether keep it or not.

Reply:
`_typing_limit` has a leading underscore, so by convention it is a subroutine that should only be called from that package. So if we don't use it in that package, IMHO we should remove it. Or alternatively use it in the places where we currently use `VNC_TYPING_LIMIT_DEFAULT`.

Reply:
`mutex_wait` is actually used: https://github.com/search?q=repo%3Aos-autoinst%2Fos-autoinst-distri-opensuse%20mutex_wait&type=code

Reply:
Yes, just because it isn't used within os-autoinst itself it doesn't mean it isn't used.

Reply:
The removal of `_typing_limit` was overlooked in 007c1837. Please remove the function

Reply:
Ok, done it!